src/main/java/chat/commands/Command.java

```diff
@@ -0,0 +1,43 @@
+package chat.commands;
+
+import chat.exceptions.ChatException;
+import chat.tasks.TaskList;
+import chat.utils.Storage;
+
+/**
+ * @author juzzztinsoong
+ */
```

Comment:
Maybe you could consider adding a class description? Noticed in several other classes as well.